codegen/src/cli.ts

```diff
@@ -55,7 +51,11 @@ program.parse();
 async function pullSchema() {
   spinner.warn('No local Xata schema found.');
   const { shouldUseCli } = await inquirer.prompt([
-    { name: 'shouldUseCli', message: 'Would you like to use the Xata CLI and clone a database?', type: 'confirm' }
+    {
+      name: 'shouldUseCli',
+      message: 'Would you like to use the Xata CLI to pull an existing database schema?',
```

Comment:
Unrelated to the scope of the PR, but this was a quick improvement.

We don't use the term `clone` anywhere. Also, you are not creating a copy of a database, but just pulling the schema.